src/crypto/litecoin.ts

```diff
@@ -85,7 +87,16 @@ class Litecoin implements ICrypto {
 
   // eslint-disable-next-line class-methods-use-this
   validateAddress(address: string): boolean {
-    return fallbackValidateAddress(address);
+    // bech32 address
+    if (address.substring(0, 3) === 'ltc') {
+      if (bech32.decodeUnsafe(address) || bech32m.decodeUnsafe(address)) {
+        return true;
+      }
+    }
+    // bs58 address
+    const res = bs58check.decodeUnsafe(address);
+    if (!res) return false;
+    return res && res.length > 3 && (res[0] === this.network.pubKeyHash || res[0] === this.network.scriptHash);
```

Comment:
Maybe digibyte and litecoin have so much in common so that it'd be beneficial to extract a superclass for these two and implement validateAddress in that superclass? Just parameterize the prefix.

Reply:
Alternatively, extract a function that digibyte and litecoin call.

Reply:
good point. I've removed bechm for litecoin and digibyte

Reply:
good point. I've done a small refactoring to make the code more concise.